auth.go

```diff
@@ -28,31 +32,85 @@ func pathLogin(b *mesosBackend) *framework.Path {
 
 // pathLogin (the method) is the "login" path request handler.
 func (b *mesosBackend) pathLogin(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
-
 	taskID := d.Get("task-id").(string)
-	if len(taskID) < 1 {
+	if !b.verifyTaskExists(taskID) {
+		return nil, logical.ErrPermissionDenied
+	}
+
+	prefix, err := taskIDPrefix(taskID)
+	if err != nil {
 		return nil, logical.ErrPermissionDenied
 	}
 
-	b.Logger().Info("LOGIN", "task-id", taskID, "RemoteAddr", req.Connection.RemoteAddr)
+	b.Logger().Info("LOGIN", "task-id", taskID, "prefix", prefix, "RemoteAddr", req.Connection.RemoteAddr)
+
+	policies, err := getTaskPolicies(ctx, req.Storage, prefix)
+	if err != nil {
+		return nil, err
+	}
 
-	// TODO: Validate the task-id and look up the associated list of policies.
 	// TODO: Make the renewal period configurable?
```

Comment:
I think the answer to this is "yes"

Reply:
Yeah. The question there is more about where we allow it to be configured (globally? per-prefix? something else?) which will affect how we handle configuration.